app/Filament/Admin/Resources/EggResource/Pages/CreateEgg.php

```diff
@@ -189,8 +190,10 @@ public function form(Form $form): Form
                                         ->hintIcon('tabler-code')
                                         ->hintIconTooltip(fn ($state) => "{{{$state}}}")
                                         ->unique(modifyRuleUsing: fn (Unique $rule, Get $get) => $rule->where('egg_id', $get('../../id')), ignoreRecord: true)
+                                        ->rules(EggVariable::$validationRules['env_variable'])
                                         ->validationMessages([
                                             'unique' => 'A variable with this name already exists.',
+                                            '*' => 'This environment variable is reserved and cannot be used.',
```

Comment:
If you leave the field empty you get the wrong validation message:
![grafik](https://github.com/user-attachments/assets/e14a3a54-1713-4b0a-926d-b996bf84ace5)